File: Iniciante/1012/1012.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner nabana = new Scanner(System.in);
        double[] d = Arrays.stream(nabana.nextLine().split(" ")).mapToDouble(Double::parseDouble).toArray();
        System.out.println("TRIANGULO: " + String.format("%.3f", (d[0]*d[2])/2));
        System.out.println("CIRCULO: " + String.format("%.3f", (3.14159*d[2]*d[2])));
        System.out.println("TRAPEZIO: " + String.format("%.3f", ((d[0]+d[1])*d[2])/2));
        System.out.println("QUADRADO: " + String.format("%.3f", (d[1]*d[1])));
        System.out.println("RETANGULO: " + String.format("%.3f", (d[0]*d[1])));
 
    }
 
}

File: Iniciante/1012/1012.cs
using System; 

class URI {

    static void Main(string[] args) { 

      string[] numeros = Console.ReadLine().Split(' ');
double A = double.Parse(numeros[0]);
double B = double.Parse(numeros[1]);
double C = double.Parse(numeros[2]);
Console.WriteLine("TRIANGULO: {0:f3}", (A*C/2d));
Console.WriteLine("CIRCULO: {0:f3}", (3.14159 * C * C));
Console.WriteLine("TRAPEZIO: {0:f3}", ((A+B)*C/2d));
Console.WriteLine("QUADRADO: {0:f3}", (B * B));
Console.WriteLine("RETANGULO: {0:f3}", (A * B));
    }

}